### PR TITLE
Cleans up the difference between Pharmacy and Chemistry access

### LINKED ISSUE #48166: Apothecary - RC announcment access
`code/__DEFINES/access.dm`
image

Well... Does it mean, that chemists have RC access too?
And should then?

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request
Fixes #48166. For the sake of clarification, Pharmacy access is meant to be for the Chemistry office in Medical, ie 'where all the meds are made'. This lets Medical Doctors and the CMO have access to the room, letting them get work done in the event Chemistry never joined/is dead/has fucked off. Chemistry access allows access to the Chemistry factory rooms (the giant fucking useless space on Meta and Box) so doctors can't just waltz into your ~~meth lab~~ synthmeat factory. I also renamed the Apothecary to the Pharmacy because what is this the 1700s?

This also updates the access on maps other than Meta and Box, future proofing them for if/when we add factory spaces.

## Why It's Good For The Game
Granular access makes it easier to decide who goes where and why.

## Changelog
:cl:
tweak: Apothecary access should now work correctly on all maps.
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
why is it called the apothecary in the first plast

**Comment by TheVekter** [comment]
I mean, we could change it to Pharmacy, but I didn't name the damn thing.

File: code/__DEFINES/access.dm [hunk 1]
```diff
@@ -79,6 +79,8 @@
 #define ACCESS_NETWORK 67
 /// Cloning room and clone pod ejection
 #define ACCESS_CLONING 68
+/// Chemistry office access
+#define ACCESS_APOTHECARY 69 ///Nice.
```
Comment (81Denton) [inline code/__DEFINES/access.dm]: nice.

**Review by 81Denton — DISMISSED** [review]

File: code/__DEFINES/access.dm (hunk 1 above)
Comment (TheVekter) [reply code/__DEFINES/access.dm]: Right?

**Comment by cacogen** [comment]
>I mean, we could change it to Pharmacy, but I didn't name the damn thing.

I think my bitching is in the scope of your PR though. I might change "chemistry" (chem factory) to chemical plant and then change apothecary back to chemistry if you don't want to

**Comment by Jordie0608** [comment]
No need for semantics, just call them
`ACCESS_CHEMISTRY_FACTORY`
and
`ACCESS_CHEMISTRY_OFFICE`

**Comment by TheVekter** [comment]
> No need for semantics, just call them
> `ACCESS_CHEMISTRY_FACTORY`
> and
> `ACCESS_CHEMISTRY_OFFICE`

This is the plan. I just need to actually make the change. I'll have it done soon.

**Comment by optimumtact** [comment]
Im quite offended you called the space useless

**Comment by TheVekter** [comment]
> Im quite offended you called the space useless

There's a special place in hell for the chem factory room in Box and as soon as I can figure out where to move it I'm doing so.

**Comment by TheVekter** [comment]
Okay assuming this passes Travis we should be good

**Comment by Jordie0608** [comment]
>> No need for semantics, just call them
>> `ACCESS_CHEMISTRY_FACTORY`
>> and
>> `ACCESS_CHEMISTRY_OFFICE`
>
>This is the plan. 

\> Does something different
I got played.

**Pull request merged**